solc/CommandLineParser.cpp

```diff
@@ -724,8 +724,8 @@ General Information)").c_str(),
 			// TODO: The type in OptimiserSettings is size_t but we only accept values up to 2**32-1
 			// on the CLI and in Standard JSON. We should just switch to uint32_t everywhere.
 			po::value<unsigned>()->value_name("n")->default_value(static_cast<unsigned>(OptimiserSettings{}.expectedExecutionsPerDeployment)),
-			"Set for how many contract runs to optimize. "
-			"Lower values will optimize more for initial deployment cost, higher values will optimize more for high-frequency usage."
+		    "The number of runs specifies roughly how often each opcode of the deployed code will be executed across the lifetime of the contract. "
+		    "Lower values will optimize more for initial deployment cost, higher values will optimize more for high-frequency usage."
```

Comment:
The tests are failing because of indentation, I think. Will push this.

Reply:
Yeah I used the editor on GitHub which did that :(